bedrock/exp/templates/exp/firefox/lockwise.de.html

```diff
@@ -0,0 +1,90 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
```

Comment:
Needs to extend the exp template?

```suggestion
{% extends "exp/base/base.html" %}
```